build.bat

```diff
@@ -1,5 +1,7 @@
 @ECHO OFF
 
+echo [   INFO] Build starting at %TIME% %DATE%
```

Comment:
I added it because when doing double builds I kept losing track of having done the second build. With the timestamp I can tell if I have or haven't. Too many shiny objects and I get distracted...lol

Plus the builds at work have them and I guess I'm just used to it.